drag_and_drop_v2/public/themes/apros.css

```diff
@@ -0,0 +1,88 @@
+.themed-xblock.xblock--drag-and-drop {
+    background: #fff;
+}
+
+/* Shared styles used in header and footer */
+
+.themed-xblock.xblock--drag-and-drop .title1 {
+    color: #555555;
+    text-transform: uppercase;
+    font-weight: bold;
+    font-style: normal;
+}
+
+/* drag-container holds the .item-bank and the .target */
+
+.themed-xblock.xblock--drag-and-drop .drag-container {
+    background: #ebf0f2;
+}
+
+.themed-xblock.xblock--drag-and-drop .item-bank {
+    border-radius: 0px;
+}
+
+/*** DRAGGABLE ITEMS ***/
+
+.themed-xblock.xblock--drag-and-drop .drag-container .option {
+    border-radius: 0px;
+    font-size: 14px;
+    background-color: #2e83cd;
+    color: #fff;
+    opacity: 1;
+}
+
+.themed-xblock.xblock--drag-and-drop .drag-container .option .numerical-input.correct .input {
+    background: #ceffce;
+    color: #087108;
+}
+
+.themed-xblock.xblock--drag-and-drop .drag-container .option .numerical-input.incorrect .input {
+    background: #ffcece;
+    color: #ad0d0d;
+}
+
+.themed-xblock.xblock--drag-and-drop .drag-container .option.fade {
+    opacity: 0.5;
+}
+
+/*** DROP TARGET ***/
+
+.themed-xblock.xblock--drag-and-drop .target {
+    background: #fff;
+}
+
+.themed-xblock.xblock--drag-and-drop .zone p {
+    font-family: Arial;
+    font-size: 16px;
+    font-weight: bold;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+/*** FEEDBACK ***/
+
+.themed-xblock.xblock--drag-and-drop .feedback {
+    border-top: solid 1px #bdbdbd;
+}
+
+.themed-xblock.xblock--drag-and-drop .popup {
+    background: none repeat scroll 0 0 #66a5b5;
+}
+
+.themed-xblock.xblock--drag-and-drop .popup .popup-content {
+    color: #ffffff;
+    font-size: 14px;
+}
+
+.themed-xblock.xblock--drag-and-drop .popup .close {
+    cursor: pointer;
+    color: #ffffff;
+    font-family: "fontawesome";
+    font-size: 18pt;
```

Comment:
These rules (and probably others in this file) are identical to the unthemed version - can you remove them from this file? (It's good that these things can be overridden by the theme, but we don't need to actually include new values for rules that aren't different here in the apros theme.)